torch/csrc/jit/python/script_init.cpp

```diff
@@ -476,6 +476,14 @@ static void setInputTensorTypes(Graph& g, const Stack& stack, bool complete) {
   auto s_iter = stack.begin();
   for (auto v : input_values) {
     AT_ASSERT(s_iter != stack.end());
+    // Leave packed param types alone. This is needed for downstream passes
```

Comment:
shall we just handle all custom classes like that here? trying to reduce amount of hard-coding